zuul.d/edpm_multinode.yaml

```diff
@@ -445,11 +442,37 @@
     parent: podified-multinode-edpm-deployment-crc-bootstrap-staging
     vars:
       cifmw_enable_virtual_baremetal_support: true
+      cifmw_config_nmstate: true
```

Comment:
I can't grok from where the NNCP for the CRC is built?
But looking at https://logserver.rdoproject.org/53/2453/620324a4416ac7f15d6a28a9178c7807ba2617e0/github-check/podified-multinode-edpm-deployment-crc-bootstrap-staging-ironic/06383e3/controller/ci-framework-data/artifacts/manifests/nmstate/crc-nncp.yaml - it does not have the additional baremetal interface.

Reply:
That part is still missing, all I'm trying to do get the nmstate part to work. Give me some more patches and I'll reach that part.